lms/djangoapps/grades/admin.py

```diff
@@ -0,0 +1,27 @@
+"""
+Django admin page for grades models
+"""
+from django.contrib import admin
+
+from config_models.admin import ConfigurationModelAdmin
+
+from lms.djangoapps.grades.models import CourseAuthorization, SubsectionGradesEnabledFlag
+from lms.djangoapps.grades.forms import CourseAuthorizationAdminForm
+
+
+class CourseAuthorizationAdmin(admin.ModelAdmin):
+    """Admin for enabling subsection grades on a course-by-course basis."""
+    form = CourseAuthorizationAdminForm
+    fieldsets = (
+        (None, {
+            'fields': ('course_id', 'enabled'),
+            'description': '''
+Enter a course id in the following form: Org/Course/CourseRun, eg MITx/6.002x/2012_Fall
+Do not enter leading or trailing slashes. There is no need to surround the course ID with quotes.
+Validation will be performed on the course name, and if it is invalid, an error message will display.
```

Comment:
This prompt is outdated and incorrect. It just needs a course_id in a form that can be handled by `course_key = CourseKey.from_string(cleaned_id)` or `course_key = SlashSeparatedCourseKey.from_deprecated_string(cleaned_id)`, and can be mapped to a course by `modulestore().has_course(course_key)`. See screenshot for devstack results.

<img width="1420" alt="screen shot 2016-08-26 at 1 51 29 pm" src="https://cloud.githubusercontent.com/assets/7373924/18015138/526bf550-6b94-11e6-8441-09b8bc216934.png">
<img width="1431" alt="screen shot 2016-08-26 at 1 51 48 pm" src="https://cloud.githubusercontent.com/assets/7373924/18015137/526a308a-6b94-11e6-9e00-6a3bf49cfe30.png">